File: mobile/lib/core/config/zona_cobertura.dart
class ZonaCobertura {
  ZonaCobertura._();

  static const double latitudMinima = 11.55;
  static const double latitudMaxima = 11.95;
  static const double longitudMinima = -86.45;
  static const double longitudMaxima = -86.05;

  static bool estaDentroDeCarazo(double? latitud, double? longitud) {
    if (latitud == null || longitud == null) return false;

    return latitud >= latitudMinima &&
        latitud <= latitudMaxima &&
        longitud >= longitudMinima &&
        longitud <= longitudMaxima;
  }
}

File: backend/WebApi.Models/ZonaCobertura.cs
namespace WebApi.Models;

public static class ZonaCobertura {
    public const decimal LatitudMinima = 11.55m;
    public const decimal LatitudMaxima = 11.95m;
    public const decimal LongitudMinima = -86.45m;
    public const decimal LongitudMaxima = -86.05m;

    public static bool EstaDentroDeCarazo(decimal? latitud, decimal? longitud) {
        if (latitud is null || longitud is null) return false;

        return latitud >= LatitudMinima && latitud <= LatitudMaxima
            && longitud >= LongitudMinima && longitud <= LongitudMaxima;
    }
}
